src/ParkingAPI.java

```diff
@@ -2,14 +2,30 @@
 
 public class ParkingAPI {
 
-    
+	public ParkingAPI() {
 
-    public ParkingAPI() {
+	}
 
-    }
+	public int getOpenSpots(ParkingLot parkingLot) {
+		Random random = new Random(parkingLot.ordinal());
+		return random.nextInt(110);
+	}
+	
+	public int getFilledSpots(ParkingLot parkingLot) {
```

Comment:
I know this is meant to be a basic api, but perhaps we should set a "variable" in each of the enums for the total number of parking lots. If you want help with figuring this out, I'd be happy to show you a bit when it comes to enums.